ovirtlib4/vms.py

```diff
@@ -43,6 +44,72 @@ def get_hosted_engine_host(self):
                 return hosts.Hosts(self.connection).get_entity_by_id(id=host_id)
         return None
 
+    @property
+    def import_vm(self):
+        """
+        Import an external VM
+
+        Example:
+            engine.vms.import_vm(
+                vm=engine.vms.import_vm.entity_type(
+                    vm=types.Vm(name="vm_name"),
+                    name="vm_name",
+                    username="admin",
+                    password="admin",
+                    provider=types.ExternalVmProviderType(external_type),
+                    url="ova:///root/file.tar.gz",  # /root/file.tar.gz found at the host_name below
+                    cluster=types.Cluster(name="cluster_name"),
+                    storage_domain=types.StorageDomain(name="storage_name"),
+                    sparse=external_vm.get('sparse', True),
+                    host=types.Host(name="host_name"),
+                )
+            )
+        """
+        return ImportVm(connection=self.connection)
+
+    @property
+    def import_template(self):
+        """
+        Import VM from template
+
+        Example:
+            engine.vms.import_template(
+                vm=engine.vms.import_template.entity_type(
+                    clone=None,
+                    cluster=None,
+                    cpu_profile=None,
+                    host=None,
+                    quota=None,
+                    storage_domain=None,
+                    template=None,
+                    url=None,
+                )
+            )
+        """
+        return ImportTemplate(connection=self.connection)
```

Comment:
These do not belong under Vms.
import_template should be under TemplateCollection and import_vm should be under DataCenterCollection or ClusterCollection or both

Reply:
you are right regarding import_template, it should sit under the 'templates' collection
but I can see Import VM at the VMs list at the WebUI, so I 'vms' collection is the right place for import_vm()
Any way I remove the imports, we handle those methods in the next patch